global.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.svg'
```

Comment:
This change was for TS to resolve svg imports when we use them as an image source